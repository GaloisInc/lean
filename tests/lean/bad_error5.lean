structure S :=
  (α : Type)
  (β : unit)

structure T (c : S)

structure U (c: S) (A : c^.α)

definition V (c : S) : S :=
{
  α := T c,
  β := by sorry
}
