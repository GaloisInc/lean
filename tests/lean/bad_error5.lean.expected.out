bad_error5.lean:9:0: warning: declaration 'V' uses sorry
